client/src/GamemasterInGame/Lobby.tsx

```diff
@@ -59,20 +59,24 @@ export function Lobby({ rolesList }: LobbyProps) {
               <BsPeopleFill />
             </Text>
           </Tabs.Trigger>
-          {!game.gameStarted && (
-            <Tabs.Trigger className="max-w-[200px] flex-1" value="roles">
-              <Text className="mr-1">
-                Roles(
-                {Object.keys(characterSelectState.selectedRoles.value).length})
-              </Text>
+          {game.gameStarted ? (
+            <Tabs.Trigger className="max-w-[200px] flex-1" value="night-order">
+              <Text className="mr-1">Night Order</Text>
               <Text color="red" asChild>
                 <FaMasksTheater />
               </Text>
             </Tabs.Trigger>
-          )}
-          {game.gameStarted && (
-            <Tabs.Trigger className="max-w-[200px] flex-1" value="nightOrder">
-              <Text className="mr-1">Night Order</Text>
+          ) : (
+            <Tabs.Trigger className="max-w-[200px] flex-1" value="roles">
+              <Text className="mr-1">
+                Roles(
+                {
+                  Object.values(
+                    characterSelectState.selectedRoles.value,
+                  ).filter((selected) => selected).length
+                }
```

Comment:
Small error was counting deselected roles in tab header.

Reply:
.filter(Boolean)